.github/workflows/nightly-short-session-e2e-tests.yml

```diff
@@ -103,16 +153,16 @@ jobs:
         shell: bash
         run: docker load -i aleph-e2e-client.tar
 
-      - name: Run finalization e2e test
+      - name: Run ${{ matrix.testcase }} e2e test
         shell: bash
         # yamllint disable rule:line-length
         run: |
           docker run --network host \
-            -e NODE_URL="wss://${{ steps.create-featurenet.outputs.ws-hostname }}.dev.azero.dev:443" \
-            -e TEST_CASES="finalization::finalization" \
+            -e NODE_URL="wss://${{ steps.create-featurenet.outputs.ws-hostname }}:443" \
+            -e TEST_CASES="${{ matrix.testcase }}" \
             -e RUST_LOG=info \
             aleph-e2e-client:latest
-        # yamllint enable rule:line-length
```

Comment:
I'd bring back that line. Besides that, lgtm.

Reply:
Right, added back.